src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.cs

```diff
@@ -255,9 +265,6 @@ public string DisplayName
         /// </summary>
         public IEnumerable<KeyValuePair<string, object?>> TagObjects
         {
-#if ALLOW_PARTIALLY_TRUSTED_CALLERS
```

Comment:
@tarekgh I removed the `SecuritySafeCritical`. Assuming this is no longer needed because we removed the `Unsafe.As` on the last PR?

Reply:
make sense. If you run the tests locally should catch any issue with the security attributes when running against NetFX.